test/e2e/hooks_test.go

```diff
@@ -805,7 +805,7 @@ spec:
         args: ["sleep", "5"]
 `).When().
 		SubmitWorkflow().
-		WaitForWorkflow(fixtures.ToBeCompleted).
+		WaitForWorkflow(fixtures.ToBeCompleted, killDuration).
```

Comment:
Timeout increased because this test is flaky:
`    hooks_test.go:808: timeout after 1m30s waiting for condition`
https://github.com/argoproj/argo-workflows/actions/runs/11922046836/job/33227659623?pr=13921